### PR TITLE
Eobs 614 set clinical review frequency task triggered

### PR DESCRIPTION
**Pull request by moderatemisbehaviour**
* **Clinical Review Frequency** task triggered after completion of **Clinical Review** task.
* Can be completed by either nurses or doctors.
* **Clinical Review Frequency** task is visible in doctor's desktop views.
* Task presents different frequencies for selection depending on patient's clinical risk.

### CODE REVIEW

File: nh_eobs_mental_health/security/ir.model.access.csv [hunk 1]
```diff
@@ -7,4 +8,14 @@ dev_access_nh_clinical_notification_clinical_review,developer:access_nh_clinical
...
 hca_access_nh_clinical_notification_clinical_review,hca:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_hca,1,1,1,0,0
-doctor_access_nh_clinical_notification_clinical_review,doctor:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_doctor,1,1,1,1,1
\ No newline at end of file
+doctor_access_nh_clinical_notification_clinical_review,doctor:access_nh_clinical_notification_clinical_review,model_nh_clinical_notification_clinical_review,nh_clinical.group_nhc_doctor,1,1,1,1,1
+
+base_access_nh_clinical_notification_clinical_review_frequency,restrict:access_nh_clinical_notification_clinical_review_frequency,model_nh_clinical_notification_clinical_review_frequency,nh_clinical.group_nhc_base,0,0,0,0,0
+nurse_access_nh_clinical_notification_clinical_review_frequency,nurse:access_nh_clinical_notification_clinical_review_frequency,model_nh_clinical_notification_clinical_review_frequency,nh_clinical.group_nhc_nurse,1,1,0,0,1
```
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/security/ir.model.access.csv]: Check ward manager and senior manager are still able to print reports.

File: nh_eobs_mental_health/tests/nh_clinical_notification_clinical_review_frequency/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# from . import test_access
```
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/tests/nh_clinical_notification_clinical_review_frequency/__init__.py]: Delete it.

File: nh_observations/nh_clinical_extension.py [hunk 3]
```diff
@@ -179,9 +179,12 @@ def trigger_notifications(self, cr, uid, values, context=None):
                 d_values = {
                     'patient_id': values.get('patient_id')
                 }
+                # Populate required observation fields for
+                # nh.clinical.notification.frequency and it's children.
                 if n['model'] in ['frequency',
                                   'select_frequency',
-                                  'weekly_frequency']:
+                                  'weekly_frequency',
+                                  'clinical_review_frequency']:
```
Comment (moderatemisbehaviour) [inline nh_observations/nh_clinical_extension.py]: Add tech debt.

Comment (moderatemisbehaviour) [inline nh_observations/nh_clinical_extension.py]: Done.

File: nh_eobs_mental_health/tests/nh_clinical_notification_clinical_review_frequency/__init__.py (hunk 2 above)
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/tests/nh_clinical_notification_clinical_review_frequency/__init__.py]: Done.

File: nh_eobs_mental_health/security/ir.model.access.csv (hunk 1 above)
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/security/ir.model.access.csv]: Checked this and (after modifying the report to show actions triggered from partials) the *clinical review* and *clinical review frequency* tasks both appeared in the *Actions Triggered* section.

**Comment by Gimpneek** [comment]
:shipit:

**Comment by Gimpneek** [comment]
super :shipit: I guess

**Comment by Gimpneek** [comment]
:shipit:  for refactoring

**Comment by Nayira** [comment]
Tests for EOBS-614, and feature bugs EOBS-732 and EOBS-740 are a Pass :shipit:

**Pull request merged**